versions.mk

```
@@ -12,4 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-GOLANG_VERSION := 1.16.0
+MDBOOK_VERSION := v0.4.5
+GOLANG_VERSION := 1.13.15
```

@jichenjc this leads to the failure.

```suggestion
GOLANG_VERSION := 1.16.0
```

---

This target has to pull the images before the build starts and the version has to match to what is used during the build:
https://github.com/kubernetes-sigs/cluster-api-provider-openstack/blob/master/Makefile#L221-L224

---

thanks!, my bad, didn't notice here, will update accordingly

---

No problem, you're welcome :)